great_expectations/jupyter_ux/__init__.py

```diff
@@ -104,15 +84,45 @@ def formatTime(self, record, datefmt=None):
                 s = self.default_msec_format % (t, record.msecs)
             return s
 
-    logger = logging.getLogger()
+    if not logger:
+        logger = logging.getLogger()
     chandler = logging.StreamHandler()
     chandler.setLevel(logging.DEBUG)
     chandler.setFormatter(Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s", "%Y-%m-%dT%H:%M:%S%z"))
     logger.addHandler(chandler)
-    logger.setLevel(logging.INFO)
-    logger.setLevel(logging.INFO)
+    logger.setLevel(logging.ERROR)
+    # logger.setLevel(logging.INFO)
     logging.debug("test")
 
     # Filter warnings
     import warnings
     warnings.filterwarnings('ignore')
+
+def list_available_data_asset_names(context, data_source_name=None):
```

Comment:
This method prints out all the datasources, generators and data asset names in the project. It replaces a more limited method above.